src/main/java/spoon/reflect/cu/CompilationUnit.java

```diff
@@ -67,6 +67,13 @@
 	 */
 	void setLineSeparatorPositions(int[] lineSeparatorPositions);
 
+	/**
+	 * Gets all binary (.class) files that corresponds to this compilation unit
+	 * and should be available after calling
+	 * {@link spoon.SpoonModelBuilder#compile(spoon.SpoonModelBuilder.InputType...)}.
+	 */
+	List<File> getExpectedBinaryFiles();
```

Comment:
I would propose to put this public method in `SpoonModelBuilder` instead.

Reply:
This method is specific to a CompilationUnit. The returned files correspond to the CompilationUnit the method was called on.